<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>156</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item>
    <widget class="QLabel" name="labelDescription">
     <property name="text">
      <string>dummy</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout" columnstretch="4,10">
      <item row="0" column="0">
       <widget class="QLabel" name="labelFormat">
        <property name="text">
         <string>Format</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinArgumentIdx">
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelArgumentIdx">
        <property name="text">
         <string>Argument index</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="labelDefaultValue">
        <property name="text">
         <string>Default value</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinDefaultValue">
        <property name="minimum">
         <number>-2147483647</number>
        </property>
        <property name="maximum">
         <number>2147483647</number>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="comboFormat">
        <item>
         <property name="text">
          <string>Up to 2 digits (4 becomes &quot;4&quot;)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Up to 3 digits (4 becomes &quot;4&quot;)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Up to 4 digits (4 becomes &quot;4&quot;)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Up to 5 digits (4 becomes &quot;4&quot;)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Up to 6 digits (4 becomes &quot;4&quot;)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>2 digits with leading zeros (4 becomes &quot;04&quot;)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>3 digits with leading zeros (4 becomes &quot;004&quot;)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>4 digits with leading zeros (4 becomes &quot;0004&quot;)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>5 digits with leading zeros (4 becomes &quot;00004&quot;)</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>6 digits with leading zeros (4 becomes &quot;000004&quot;)</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
